shoes-swt/spec/shoes/swt/app_spec.rb

```diff
@@ -122,4 +128,20 @@
       it_behaves_like 'reports client area dimensions'
     end
   end
+
```

Comment:
Here are the only two specs I tried to add to spec this out. I've commented out the second one because it has been tough to figure out because this whole system color thing requires a full front and backend.

I could stub in something that acts like a `Shoes::Color`  on the double . . . is that useful though?